lib/components/elements/tables/row_component.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= tag.tr id: do %>
+  <% if label.present? %>
+    <th class="col-3" scope="row">
+      <%= label %>
+      <%#= render Elements::TooltipComponent.new(target_label: label, tooltip:) %>
```

Comment:
Intentional commenting?

Reply:
Yes, we don't have tooltips in this app, so I didn't port the component.